src/Containers/RideProgressCard/RideProgressCard.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { View, StyleSheet } from "react-native";
+import ProgressLine from "src/Components/ProgressLine";
+import Cell from "./Components/Cell";
+
+type Props = {
+  origin: {
+    time: string;
+    day: string;
+    city: string;
+    address?: string;
+  };
+  destination: {
+    time: string;
+    day: string;
+    city: string;
+    address?: string;
+  };
+  currentProgress?: number;
+};
+
+class RideProgressCard extends React.PureComponent<Props> {
+  public render() {
+    return (
+      <View style={styles.container}>
+        {this.renderTimeAndDate()}
+        <ProgressLine currentProgress={this.props.currentProgress} />
+        {this.renderCities()}
+      </View>
+    );
+  }
+
+  private renderTimeAndDate = () => {
+    const { origin, destination } = this.props;
+    return (
+      <View style={styles.timeAndDateContainer}>
+        <Cell mainText={origin.time} secondaryText={origin.day} />
```

Comment:
There is a problem with your composition. It is really strange.

You've created a separate component for Cell. 

1. Name is too generic, I can not understand from it's name what part of the interface it implements by looking at RideProgressCard design
2. It is so small component, that's even strange why you've choose it to move to a separate component
3. By looking at RideProgressCard I can see, that 

```
      <View style={styles.citiesContainer}>
        <Cell mainText={origin.city} secondaryText={origin.address} />
        <Cell mainText={destination.city} secondaryText={destination.address} />
      </View>
```

is used twice in the same fashion (just the styles) are different. We use composition to eliminate the repeating. 

In [this thread](https://trucknet-team.slack.com/archives/CFDNTJQ7N/p1575541931175500) we were speaking that, if you use render method inside component it usually means it is the first candidate to be moved to a separate component.